src/otx/recipe/segmentation/dino_v2_seg.yaml

```diff
@@ -0,0 +1,130 @@
+# @package _global_
+defaults:
+  - override /base: segmentation
+  - override /callbacks: segmentation
+  - override /data: segmentation
+  - override /model: mmseg
+data:
+  train_subset:
+    batch_size: 8
+    transforms:
+      - type: LoadImageFromFile
+      - reduce_zero_label: true
+        type: LoadAnnotations
+      - keep_ratio: false
+        ratio_range:
+          - 0.5
+          - 2.0
+        scale:
+          - 640
+          - 640
+        type: RandomResize
+      - cat_max_ratio: 0.75
+        crop_size:
+          - 560
+          - 560
+        type: RandomCrop
+      - prob: 0.5
+        type: RandomFlip
+      - type: PhotoMetricDistortion
+      - type: PackSegInputs
+  val_subset:
+    batch_size: 1
+    transforms:
+      - type: LoadImageFromFile
+      - keep_ratio: false
+        scale:
+          - 560
+          - 560
+        type: Resize
+      - reduce_zero_label: true
+        type: LoadAnnotations
+      - type: PackSegInputs
+  test_subset:
+    batch_size: 1
+    transforms:
+      - type: LoadImageFromFile
+      - keep_ratio: false
+        scale:
+          - 560
+          - 560
+        type: Resize
+      - reduce_zero_label: true
+        type: LoadAnnotations
+      - type: PackSegInputs
+model:
+  otx_model:
+    config:
+      backbone:
+        name: dinov2_vits14_reg
+        freeze_backbone: true
+        out_index:
+          - 8
+          - 9
+          - 10
+          - 11
+        type: DinoVisionTransformer
+      data_preprocessor:
+        bgr_to_rgb: true
+        mean:
+          - 123.675
+          - 116.28
+          - 103.53
+        pad_val: 0
+        seg_pad_val: 255
+        size:
+          - 560
+          - 560
+        std:
+          - 58.395
+          - 57.12
+          - 57.375
+        test_cfg:
+          size_divisor: 14
+        type: SegDataPreProcessor
+      decode_head:
+        init_cfg:
+          checkpoint: https://dl.fbaipublicfiles.com/dinov2/dinov2_vits14/dinov2_vits14_ade20k_linear_head.pth
+          type: Pretrained
+          prefix: decode_head
+        type: FCNHead
+        in_channels:
+          - 384
+          - 384
+          - 384
+          - 384
+        in_index:
+          - 0
+          - 1
+          - 2
+          - 3
+        input_transform: resize_concat
+        channels: 1536
+        kernel_size: 1
+        num_convs: 1
+        concat_input: false
+        dropout_ratio: -1
+        num_classes: 2
+        norm_cfg:
+          type: SyncBN
+          requires_grad: true
+        align_corners: false
+        loss_decode:
+          type: CrossEntropyLoss
+          use_sigmoid: false
+          loss_weight: 1.0
+      pretrained: null
+      test_cfg:
+        mode: whole
```

Comment:
I investigated the configuration in this notebook, https://github.com/facebookresearch/dinov2/blob/main/notebooks/semantic_segmentation.ipynb.
However, what's the reason for the difference in `test_cfg`?
![image](https://github.com/openvinotoolkit/training_extensions/assets/26541465/14ca9bde-86bc-421a-b0ed-6b316f7e6a3b)
You used `mode: whole` but the notebook used `mode: slide`.

Reply:
I aligned testing configuration with other our segmentation models. There shouldn't be any differences in final metric. I also aligned the resolution to our standard resolution that we use in Geti.

Reply:
<img width="550" alt="image" src="https://github.com/openvinotoolkit/training_extensions/assets/26541465/15680f63-3d28-490d-b922-f4a515a9b4ce">

https://mmsegmentation.readthedocs.io/en/latest/advanced_guides/models.html#prediction-modes

I can see here that `test_cfg` really affects to the model `forward()` in the prediction. It seems being able to affect the final metric.

For VisionTransformer, input size is more important than ConvNet. This is because the input size affects to the token sequence length and it affects to the both sides, model accuracy and performance:

1) Self attention computation is `O(n^2)`, `n` is token sequence length.
2) If the pretrained model had been training for `n` tokens, but you gave `m > n` tokens after that, the position embedding for `k=n+1,...,m` might not have been learnt well.

Reply:
Okay, let me check if there will be the difference

Reply:
So, considering my experiments, I stick to current settings